From 0104bc562b59ee26981fa93beb8d66e1fbe2e453 Mon Sep 17 00:00:00 2001
From: Ondrej Jirman <megous@megous.com>
Date: Fri, 12 Apr 2019 13:24:26 +0200
Subject: [PATCH 12/29] bluetooth: hci_bcm: Give more time to come out of reset

Some devices need more time to come out of reset (eg. BCM4345).
Increase the post-reset delay. I don't have datasheet, so rather
be safe than to get intermittent failures during probe.

Signed-off-by: Ondrej Jirman <megous@megous.com>
---
 drivers/bluetooth/hci_bcm.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/bluetooth/hci_bcm.c b/drivers/bluetooth/hci_bcm.c
index 8905ad2..8f56df5 100644
--- a/drivers/bluetooth/hci_bcm.c
+++ b/drivers/bluetooth/hci_bcm.c
@@ -260,7 +260,7 @@ static int bcm_gpio_set_power(struct bcm_device *dev, bool powered)
 	}
 
 	/* wait for device to power on and come out of reset */
-	usleep_range(10000, 20000);
+	usleep_range(50000, 60000);
 
 	dev->res_enabled = powered;
 
-- 
2.7.1

